uniqueids/admin.py

```diff
@@ -1,6 +1,40 @@
 from django.contrib import admin
 
-# Register your models here.
 from .models import Record
+from .tasks import send_personnel_code
 
-admin.site.register(Record)
+
+class RecordAdmin(admin.ModelAdmin):
+    list_display = [
+        "id", "identity", "write_to", "created_at", "updated_at"]
+    list_filter = ["write_to", "created_at"]
+    search_fields = ["identity", "write_to"]
+    actions = ["resend_personnel_code"]
+
+    def resend_personnel_code(self, request, queryset):
+        created = 0
+        skipped = 0
+        for record in queryset:
+            if record.write_to != "personnel_code":
```

Comment:
I think you can just do `for record in queryset.exclude(write_to='personnel_code')` You won't have your count though.